helpers/getFileIconType.js

```diff
@@ -0,0 +1,103 @@
+export function getFileIconTypeFromExtension(extensionString) {
+	var iconType;
```

Comment:
Instead of using a `var` here (which the linter is complaining about), could you just return from the `case` statements?